### ISSUE TITLE
Cetus Pro 1.2.1 different build times?

### ISSUE THREAD

**Issue by krisha** [BETAFPV/BETAFPV.github.io#12]
Got a new Cetus Pro [and have an issue with Radiomaster T8 Lite V2 (binding ok, channels ok, OSD working, but doesn't arm)]. 

Figured out it was shipped with 1.2.1 but build time was ~22-04-2022.

The 1.2.1 release here has a build time of 2021.

Are there different builds with same version?

**Comment by a-silent** [comment]
I have the same question, I updated my drone to the firmware of 2021 and it's terrible, I want to return the firmware of 2022(22-04-2022) but I can't find it

**Comment by pranas** [comment]
@krisha @a-silent Did you resolve the issue? Were you able to arm Cetus Pro with T8 Lite V2 at any point? BetaFPV support recommended using 1.2.2 firmware but I can't get it to arm with that firmware either (it didn't ever arm for me).

**Comment by krisha** [comment]
@pranas No, I returned it and bought another tiny whoop from a different brand.

**Comment by a-silent** [comment]
@pranas I wrote to the support and they gave me firmware for my flight controller and it resolved my issue

**Comment by pranas** [comment]
> @pranas I wrote to the support and they gave me firmware for my flight controller and it resolved my issue

Do you use Radiomaster T8 Lite V2? Can you share the firmware? They recommended me to use 1.2.2, but I still cannot arm with that firmware. I think there's a compatibility issue with my radio, and I'm looking for another radio to test with.

**Comment by a-silent** [comment]
no, I use native radio from box (LiteRadio 2)
https://drive.google.com/file/d/1gaDAxVUBTDBqFipBpLPlWyg9V4TKVa3V/view?usp=sharing
this firmware for flight controller only version 1.3

**Comment by pranas** [comment]
For whoever will run into this issue as well: Cetus Pro is not compatible with Radiomaster T8 Lite V2. It will bind, but none of the switches (including the arm) will work. I tried the BetaFPV controller on the same drone, and it worked just fine. I'm guessing the channel values for the switch positions are different on T8 Lite, and since there's no way to configure Cetus Pro (at least on the drone that I have, the BetaFPV configurator doesn't work), you can't do anything with it.
